db/migrate/20230309002556_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :post_id
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Same here, Please create this reference between the users table and the posts table as `author_id`, not `user_id` as you did here.

- [ ] Please add a reference (foreign key) to address the relationship between the posts table and the likes table because you added a normal column `post_id` so kindly delete it and add the following line at the end.
```ruby
t.references :post, foreign_key: true
```